lib/tutorials/caching.md

```diff
@@ -0,0 +1,187 @@
+## Client side caching
+
+Http protocol provides several mechanisms how to cache resources in browser via http headers. To decide which headers are suitable for your use case check Google developers [guide](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching) or other [resources](https://www.google.com/search?q=HTTP+caching). This tutorial provides overview how to use these techniques in hapi.
```

Comment:
> mechanisms how to cache resources in browser via http headers
maybe change to "different Http headers to control how browsers cache resources."